packages/arb-token-bridge-ui/src/components/common/SidePanel.tsx

```diff
@@ -21,36 +21,92 @@ export const SidePanel = ({
   panelClassNameOverrides = '',
   scrollable = true
 }: SidePanelProps) => {
+  const [open, setOpen] = useState(false)
+  const [isClosing, setIsClosing] = useState(false)
+
+  // When user refreshes the page with the panel open, the query param starts as true
```

Comment:
Do we want transition when user refresh the page? I'd expect transition only when we change state

Reply:
When there's no transition at all then the backdrop is at 100% opacity. Change its classNames doesn't affect this either